front/app/containers/Admin/pages-menu/breadcrumbs.ts

```diff
@@ -0,0 +1,7 @@
+import messages from './messages';
+
+export const pagesAndMenuBreadcrumb = {
+  label: messages.pagesAndMenuTitle,
+  linkTo: '/pages-and-menu',
+};
+export const homeBreadcrumb = { label: messages.homeTitle, linkTo: '/home' };
```

Comment:
These links are not working. Should have admin in front of them. /home should become whatever is in 1008. Currently /home_page, but /homepage is probably nicer.

Reply:
The links should be considered placeholders, since we're working on a feature branch and have ~5 open PRs to merge into it I think it makes sense to figure out the navigation once we've got all the functionality in place. This branch doesn't have the other components in it, so the links wouldn't go anywhere even if they were accurate